docs/release-notes/8.0.32-24.2.md

```diff
@@ -0,0 +1,39 @@
+# Percona XtraDB Cluster 8.0.32-24.2 (2023-05-)
+
+<style>
+    table {
+        width=50%;
+        font-family: Poppins;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+</style>
+
+| Release date | May , 2023 |
+|---|---|
+| Install instructions | [Install Percona XtraDB Cluster](https://www.percona.com/doc/percona-xtradb-cluster/8.0/install/index.html) |
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+## Release highlights
+
+This release of Percona XtraDB Cluster 8.0.32-24 includes the fix for [PXC-4211](https://jira.percona.com/browse/PXC-4211).
+
+## Bug fixes
+
+* [PXC-4211](https://jira.percona.com/browse/PXC-4211): The server exited on the binary log rotation.
+
+* [PXC-4217](https://jira.percona.com/browse/PXC-4217): The cluster can intermittently abort a node on an insert query.
```

Comment:
There is one more in this series: PXC-4222
All of them are duplicates.

Reply:
Added.